jobserver/urls.py

```diff
@@ -337,9 +336,17 @@
     path("ui-components/", components),
     path("users/", include(user_urls)),
     path("workspaces/", yours.WorkspaceList.as_view(), name="your-workspaces"),
-    path("__debug__/", include(debug_toolbar.urls)),
-    path("<str:project_slug>/", include(project_urls)),
-] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+]
+
+if settings.DEBUG:  # pragma: nocover
+    urlpatterns += [
+        path("__debug__/", include("debug_toolbar.urls")),
+    ] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
I think we want the static files in both production and development?

Reply:
Of course we do! Thank you for spotting this.